poetry/publishing/uploader.py

```diff
@@ -272,7 +272,7 @@ def _upload_file(
                         )
                     )
                     bar.finish()
-                elif resp.status_code == 301:
+                elif resp.status_code in (301, 308):
```

Comment:
Since we're dropping Python 2 support, let's take advantage of the enums in https://docs.python.org/3/library/http.html#http.HTTPStatus instead of using the raw status codes.